plutus-benchmark/script-contexts/test/checkScriptContextEqualityTerm-20.budget.golden

```diff
@@ -1,2 +1,2 @@
-({cpu: 603616251
-| mem: 2466446})
\ No newline at end of file
+({cpu: 603524251
```

Comment:
@michaelpj This suggests a potential improvements to the compiler - A data constructor application should be considered pure if all arguments are pure.

Reply:
Yeah, I commented on this in the other PR too.